app/vue/package.json

```diff
@@ -70,17 +70,22 @@
     "url-loader": "^0.6.2",
     "util-deprecate": "^1.0.2",
     "uuid": "^3.1.0",
-    "vue": "^2.5.2",
-    "vue-hot-reload-api": "^2.2.0",
-    "vue-loader": "^12.2.1",
+    "vue-hot-reload-api": "^2.1.1",
```

Comment:
It should be `2.2.0`

Reply:
Please run `yarn install` to update lockfile after you fix it

Reply:
Not a deal-breaking for this PR, TBH.